composer.json

```diff
@@ -28,42 +27,43 @@
       "homepage": "https://github.com/browscap/browscap/graphs/contributors"
     }
   ],
+  "homepage": "https://github.com/browscap/browscap",
+  "support": {
+    "issues": "https://github.com/browscap/browscap/issues",
+    "source": "https://github.com/browscap/browscap"
+  },
   "require": {
-    "php": "^7.1.3",
+    "php": "^7.4.3",
```

Comment:
We could probably go to PHP `^8.1.0` now I think?

Reply:
For Windows there not an installable verison with PHP 8.1 at the moment, maybe next week. I upgraded my system to PHP 8 today

Reply:
I created issue #2547 for this